reflex/utils/exec.py

```diff
@@ -176,7 +179,6 @@ def output_system_info():
         dependencies.extend(
             [
                 f"[NVM {constants.NVM_VERSION} (Expected: {constants.NVM_VERSION}) (PATH: {constants.NVM_PATH})]",
-                f"[Bun {prerequisites.get_bun_version()} (Expected: {constants.BUN_VERSION}) (PATH: {constants.BUN_PATH})]",
```

Comment:
bit of a sledgehammer approach, but lets take it and get this PR in and i'll fix it in that followup PR i already have open.